tests/conversion

```diff
@@ -6,6 +6,13 @@ set -eu
 
 install_deps jq unzip
 
+# Install virt-v2v. On older Ubuntu versions, this package is available
+# under libguestfs-tools, but we do not want that as virt-v2v version
+# is too old anyway.
+if apt-cache show virt-v2v > /dev/null; then
```

Comment:
`apt-cache show` only works if a prior `apt-get update` was run. Since you've called `install_deps` before, you have it implicitly but that feels brittle.

As such, I'd prefer if we hardcode the installation when:

```
# XXX: virt-v2v is recent enough in 24.04 onward. For older releases, simply skip the v2v conversion test.
. /etc/os-release
if dpkg --compare-versions "${VERSION_ID}" ge 24.04; then
      install_deps jq unzip virt-v2v
else
      install_deps jq unzip
fi
```

Reply:
Thanks, a very nice way of loading release variables that I never thought about :)